src/Libraries/DesignScriptBuiltin/DesignScriptBuiltin.csproj

```diff
@@ -9,7 +9,8 @@
     <RootNamespace>Builtin</RootNamespace>
     <AssemblyName>DesignScriptBuiltin</AssemblyName>
     <DocumentationFile>$(OutputPath)\$(UICulture)\DesignScriptBuiltin.xml</DocumentationFile>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+      <GenerateDocumentationFile>True</GenerateDocumentationFile>
```

Comment:
if "GenerateDocumentationFile" should be true for all projects, then just add it to the CS_SDK.props file